lib/mosaic-routing/src/main/java/org/eclipse/mosaic/lib/routing/pt/PtRouting.java

```diff
@@ -96,7 +100,11 @@ public void initialize(CPublicTransportRouting routingConfiguration, File config
                 .putObject("graph.location", baseDirectory.resolve("ptgraph").toAbsolutePath().toString()) //
                 .putObject("datareader.file", baseDirectory.resolve(routingConfiguration.osmFile).toAbsolutePath().toString())
                 .putObject("gtfs.file", baseDirectory.resolve(routingConfiguration.gtfsFile).toAbsolutePath().toString())
-                .setProfiles(Collections.singletonList(new Profile("foot").setVehicle("foot")));
+                .putObject("graph.encoded_values", "foot_access, foot_priority, foot_average_speed")
+                .setProfiles(Collections.singletonList(new Profile("foot").setCustomModel(new CustomModel()
```

Comment:
How does this Profile differ from the profiles we now define as RoutingProfiles?

Reply:
The graphhopper internal `Profile` just stores the name and some properties for each profile. Our `RoutingProfile` additionally stores the instances to the individual encodings, since I want to have quick access to them.